app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
+    <string name="pref_prefer_advanced_editor_summary">Opens advanced mode when adding/editing servers</string>
+    <string name="pref_sort_server_list">Sorting of servers</string>
+    <string name="pref_sort_server_list_summary">By name</string>
     <string name="pref_gesture">Gestures</string>
-    <string name="pref_single_tap">Single Tap</string>
-    <string name="pref_two_finger_tap">2 Finger Tap</string>
+    <string name="pref_single_tap">Single tap</string>
+    <string name="pref_two_finger_tap">Two-finger Tap</string>
     <string name="pref_double_tap">Double Tap</string>
-    <string name="pref_long_press">Long Press</string>
-    <string name="pref_swipe1">1 Finger Swipe</string>
-    <string name="pref_swipe2">2 Finger Swipe</string>
-    <string name="pref_drag">Drag (Long Press then Swipe)</string>
+    <string name="pref_long_press">Long press</string>
+    <string name="pref_swipe1">1 Finger swipe</string>
+    <string name="pref_swipe2">2 Finger swipe</string>
+    <string name="pref_drag">Drag (long press then swipe)</string>
     <string name="pref_mouse">Mouse</string>
-    <string name="pref_mouse_passthrough">Mouse Passthrough</string>
-    <string name="pref_mouse_passthrough_summary_off">Mouse events are used locally for gestures</string>
-    <string name="pref_mouse_passthrough_summary_on">Mouse events are sent directly to Server</string>
-    <string name="pref_hide_local_cursor">Hide Local (Android) Cursor</string>
+    <string name="pref_mouse_passthrough">Mouse passthrough</string>
+    <string name="pref_mouse_passthrough_summary_off">Using mouse events for gestures on Android</string>
+    <string name="pref_mouse_passthrough_summary_on">Sending all mouse events to server</string>
+    <string name="pref_hide_local_cursor">Hide local (Android) cursor</string>
     <string name="pref_vk">Virtual Keys</string>
     <string name="pref_vk_show_all">Show all keys</string>
-    <string name="pref_vk_open_with_keyboard">Show when Keyboard is open</string>
-    <string name="pref_km">Key Mappings</string>
-    <string name="pref_km_right_alt_to_super">Right Alt to Super</string>
-    <string name="pref_km_language_switch_to_super">Language Switch to Super</string>
+    <string name="pref_vk_open_with_keyboard">Show when keyboard is open</string>
+    <string name="pref_km">Shortcuts</string>
+    <string name="pref_km_right_alt_to_super">Right Alt → Super modifier</string>
+    <string name="pref_km_language_switch_to_super">Language switch → Super modifier</string>
     <string name="pref_km_back_to_escape">Back to Escape</string>
-    <string name="pref_km_back_to_escape_summary">Navigation Bar is not affected by this</string>
-    <string name="pref_fullscreen">Use Fullscreen</string>
-    <string name="pref_enable_pip">Enable Picture-in-Picture</string>
+    <string name="pref_km_back_to_escape_summary">The nav bar is unaffected by this</string>
```

Comment:
```suggestion
    <string name="pref_km_back_to_escape_summary">Navigation bar is unaffected by this</string>
```
We should avoid unnecessary short word, as it might complicate translations.